bedrock/firefox/templates/firefox/firstrun/learnmore/yahoo/base.html

```diff
@@ -0,0 +1,81 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
+{% block page_og_title %}
+  {{_('Choose the independent browser')}}
+{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}
+  {{_('Firefox is non-profit, non-corporate, non-compromised. Choosing Firefox isn’t just choosing a browser. It’s a vote for personal freedom.')}}
+{% endblock %}
+
+{#- Override <meta property="og:url"> for social share -#}
+{% block page_og_url %}{{ url('firefox') }}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_firstrun_learnmore_yahoo_search' %}
+{% endblock %}
+
+{% block optimizely %}
+  {% if waffle.switch('firefox-firstrun-learnmore-optimizely') %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block site_header %}
+<header id="masthead">
+  <h2>
+    <a href="{{ url('firefox.new') }}">
+      {{ high_res_img('firefox/firstrun/learnmore/yahoo/firefox-logo.png', {'alt': 'Firefox', 'width': '97', 'height': '100'}) }}
+    </a>
+  </h2>
+  <div id="tabzilla">
+    <a href="{{ url('mozorg.home') }}" data-link-type="nav" data-link-name="tabzilla">Mozilla</a>
+  </div>
+</header>
+{% endblock %}
+
+{% block content %}
+<main role="main" class="container">
+  {% block main_heading %}
+    <h1>{{ _('Search and see, with Yahoo Search.') }}</h1>
```

Comment:
It's kind of a preference thing, but when very little is changing in the template (like just the heading), you could use a conditional and context data ([example](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/views.py#L326-L349)) to create 1 HTML file instead of 3. Would also make implementing the winning headline a touch easier (fewer templates to shuffle around).

Just a suggestion for next time. :grin: